libbeat/docs/filteringconfig.asciidoc

```diff
@@ -0,0 +1,63 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/filteringconfig.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[[configuration-filter]]
+=== Filter
+
+You can set options in the `filter` section of the {beatname_uc} config file to reduce the number of fields that are 
+exported by the Beat. See <<exported-fields>> for the full list of possible fields.
+
+The filter actions that you specify are applied to each event before it's sent to the defined output. You can define
+multiple filter actions. The filter actions are executed in the order that they're defined in the config file.
+
+==== Filter Options
+
+You can specify the following options under the `filter` section:
+
+[[include-fields]]
+===== include_fields
+
+The `include_fields` action specifies the list of fields to export. You can define full nested maps as fields. 
```

Comment:
> You can define full nested maps as fields.

How? -- I think this is done using dot-notation to specify the full name of the field (like `fieldA.nestedFieldB.nestedFieldC` or `dns.question.name`).